src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends WordListItem{
```

Comment:
```suggestion
public class Deadline extends WordListItem {
```
Be careful with the spacing requirements for Egyptian style brackets!